includes/class-performance-wizard-data-source-themes-and-plugins.php

```
@@ -48,11 +48,62 @@ public function get_data(): string {
+					if ( is_wp_error( $unzip ) ) {
+						$plugins_data['download_error'] = $unzip->get_error_message();
+					}
+				} else {
+					$plugins_data['download_error'] = $downloaded->get_error_message();
+				}
+
+				// Traverse the downloaded files and gather up the path and source for all html, php, css and js files.
+				$files = new RecursiveIteratorIterator(
+					new RecursiveDirectoryIterator( '/tmp/' . $plugin_slug )
+				);
+				include_once ABSPATH . 'wp-admin/includes/file.php';
```

PHPStan (2.1.40)

[error] Constant ABSPATH not found.
Learn more at https://phpstan.org/user-guide/discovering-symbols

(constant.notFound)